internal/sdkprovider/service/kafkaschema/kafka_schema_test.go

```diff
@@ -269,6 +295,62 @@ data "aiven_kafka_schema" "schema" {
 }`, os.Getenv("AIVEN_PROJECT_NAME"), name, name)
 }
 
+// testAccKafkaSchemaProtobufResource is a test resource for Protobuf schema Kafka Schema resource.
+func testAccKafkaSchemaProtobufResource(name string) string {
+	return fmt.Sprintf(`
+data "aiven_project" "foo" {
+  project = "%s"
+}
+
+resource "aiven_kafka" "bar" {
+  project                 = data.aiven_project.foo.project
+  cloud_name              = "google-europe-west1"
+  plan                    = "startup-2"
+  service_name            = "test-acc-sr-%s"
+  maintenance_window_dow  = "monday"
+  maintenance_window_time = "10:00:00"
+
+  kafka_user_config {
+    schema_registry = true
+
+    kafka {
+      group_max_session_timeout_ms = 70000
+      log_retention_bytes          = 1000000000
+    }
+  }
+}
+
+resource "aiven_kafka_schema_configuration" "foo" {
+  project             = aiven_kafka.bar.project
+  service_name        = aiven_kafka.bar.service_name
+  compatibility_level = "BACKWARD"
+}
+
+resource "aiven_kafka_schema" "foo" {
+  project      = aiven_kafka_schema_configuration.foo.project
+  service_name = aiven_kafka_schema_configuration.foo.service_name
+  subject_name = "kafka-schema-%s"
+  schema_type  = "PROTOBUF"
+
+  schema = <<EOT
+syntax = "proto3";
+
+message Example {
+  int32 test = 5;
+}
+
+EOT
+}
+
+data "aiven_kafka_schema" "schema" {
+  project      = aiven_kafka_schema.foo.project
+  service_name = aiven_kafka_schema.foo.service_name
+  subject_name = aiven_kafka_schema.foo.subject_name
+
+  depends_on = [aiven_kafka_schema.foo]
+}`, os.Getenv("AIVEN_PROJECT_NAME"), name, name)
```

Comment:
Is it possible to extend existing test with json schema (to not create an extra service)?

Reply:
should be, I'll update the PR